x509util/files.go

```diff
@@ -52,12 +52,22 @@ func ReadPossiblePEMURL(target, blockname string) ([]byte, error) {
 	return dePEM(data, blockname), nil
 }
 
-func dePEM(data []byte, blockname string) []byte {
+func dePEM(data []byte, blockname string) [][]byte {
+	var results [][]byte
 	if strings.Contains(string(data), "BEGIN "+blockname) {
-		block, _ := pem.Decode([]byte(data))
-		if block != nil && block.Type == blockname {
-			return block.Bytes
+		rest := data
+		for {
+			var block *pem.Block
+			block, rest = pem.Decode(rest)
+			if block == nil {
+				break
```

Comment:
Do you care if there's still stuff in `rest` here?

Reply:
Yeah, this is in a loop that reads multiple PEM blocks from one source.